### PR TITLE
Fix #2833: Add bazel util networking

### LINKED ISSUE #2833: Add missing named argument
**Describe the bug**
We are passing arguments without specifying their names.

**Expected behavior**
Look for the below-mentioned test files and add all the missing named arguments 

- [x] AdministratorControlsActivityTest (@dishantvyas15 )
- [x] AppVersionActivityTest (@darshanhande11)
- [x] CompletedStoryListActivityTest (@techjd)
- [x] FAQListFragmentTest (@ksog66)
- [x] HelpFragmentTest (@ravi9607)
- [x] RecentlyPlayedFragmentTest (@techjd)
- [x] OngoingTopicListActivityTest (@nlok5923)
- [x] OptionsFragmentTest  (@yashyon)
- [x] ProfileChooserFragmentTest (@mrwhoknows55)
- [x] ProfileProgressFragmentTest (@Arnold2381)
- [x] BindableAdapterTest (@jkugsiya)
- [x] ProfileEditActivityTest (@ArpitShukIa)
- [x] ProfileListFragmentTest (@Naman-1234)
- [x] SplashActivityTest (@darshanhande11)
- [x] StoryActivityTest (@jkugsiya)
- [x] StoryFragmentTest (@ChiragKalra)
- [x] DragDropTestActivityTest (@sakshampruthi)
- [x] ImageRegionSelectionInteractionViewTest (@Karanjot-singh)
- [x] TopicTestActivityForStoryTest (@nlok5923)
- [x] TopicInfoFragmentTest (@darshanhande11)
- [x] TopicLessonsFragmentTest (@Amar-2003)
- [x] TopicPracticeFragmentTest (@Karanjot-singh)
- [x] TopicRevisionFragmentTest (@justdvnsh)

**Additional context**
What is Named Arguments? 
https://kotlinlang.org/docs/functions.html#named-arguments

**How to work on this issue**
Comment below "I want to work on [FILE_NAME]", mention the file name from the above list you want to work on. 
Please work on only one file at a time. 

If you are a First Time Contributor to Oppia-Android, Start [here](https://github.com/oppia/oppia-android/wiki#onboarding-instructions).

**Note: Don't assign this issue to anyone as this issue should be done in a multiple PR fashion "single file, single PR"**

### LINKED ISSUE #2883: Create a BUILD.bazel file for the networking package in the utility module
This task is part of the build system migration from Gradle to Bazel. 

## Prerequesites

- Make sure you are able to build Oppia using Bazel
- Follow the [Oppia Bazel Setup instructions](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Setup-Instructions) to setup Bazel and learn the terminology.
- Read the [Gradle Bazel Migration Best Practies and FAQ](https://github.com/oppia/oppia-android/wiki/Gradle--Bazel-Migration-Best-Practices-and-FAQ) for more context and see if there's an answer there to any question you have.

## Task

1. Create a BUILD.bazel file in

`src/main/java/org/oppia/android/util/networking/`

2. Define the kt_android_library library

Based on the imports in the Kotlin files, add the required third_party dependencies to make the libraries build. 

Run
```
bazel build :all
```
inside the directory, to verify that the libraries build.

Check the Useful links section for a list of third_party dependencies.

You can start with the following stub

```
load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")

kt_android_library(
    name = "network_conecction_util",
    srcs = [
        "NetworkConnectionUtil.kt",
    ],
)
```

3. Add the directory to the MIGRATED_PROD_FILES of the module's root BUILD.bazel file

4. Add the libraries to the list of deps dependencies in the targets that require the library

You will have to set the apropriate visbility. See the list of visibilities in the link below.

5. Clean up any unused deps dependencies

6. Verify that the app builds

## Useful links

- [BUILD.bazel files](https://github.com/oppia/oppia-android/search?q=filename%3ABUILD.bazel&type=code)
- [Visibilities](https://github.com/oppia/oppia-android/blob/76dc13dba3ad77aecd2218a5372949d6551f12fe/BUILD.bazel)
- [third_party dependencies](https://github.com/oppia/oppia-android/blob/a70d9b016f0d77dd30c4939041f82ea02ee28d1d/third_party/versions.bzl)

### PR DESCRIPTION
**Pull request by jonathanalvares9009**
Package for networking utilities
FIxes #2883.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by jonathanalvares9009** [comment]
@anandwana001 can you assign @BenHenning this PR for him to review it?

**Comment by anandwana001** [comment]
@jonathanalvares9009 could you please correct the issue number 2919 is a pr not the issue.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp can you assign @BenHenning to review this PR?

File: utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""
```
Comment (fsharpasharp) [inline utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel:1]: Consider rephrasing this sentence, since it seems a bit tautological.

**Review by fsharpasharp — CHANGES_REQUESTED** [review]
Thanks! Looks good.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp made the change you requested.

File: utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel (hunk 1 above)
Comment (fsharpasharp) [reply utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel:1]: I think something like 

"Package for networking utilities."

is enough.

**Comment by fsharpasharp** [comment]
Can you check and fix the linting issue?

```
********************************
Checking Bazel file formatting
********************************
utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel # reformat
********************************
Buildifier issue found.
Please fix the above issues.
```

**Comment by jonathanalvares9009** [comment]
@fsharpasharp I tried running ktlint -F **file src** but it doesn't make the changes can you help me?

**Comment by jonathanalvares9009** [comment]
@anandwana001 my commit is failing the lint test but I ran ktlint -f **file_name** but it showed no errors n there wasn't a difference when I ran git diff. COuld you help me?

**Comment by fsharpasharp** [comment]
@jonathanalvares9009 the ktlinter won't work on BUILD.bazel files. Try to put the visibility field before the deps field.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp thanks passed ktlint test.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp can you review my PR?

**Comment by fsharpasharp** [comment]
Could you amend the description with something similar to my suggestion? We want to describe the package rather than the "implementation" of the library.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp is it ok now?

**Comment by jonathanalvares9009** [comment]
@fsharpasharp and @BenHenning please review my PR.

**Comment by fsharpasharp** [comment]
> @fsharpasharp is it ok now?

I don't see an update to the comment. Update the above thread to make this conversation easier to follow.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp I'm confused what am I supposed to update now? I update the PR description to **Package for networking utilities** and the comment in the bazel file to **This file contains the dependecies needed by the files under the networking package to run**. Can you please tell me what to change?

**Comment by fsharpasharp** [comment]
> @fsharpasharp I'm confused what am I supposed to update now? I update the PR description to **Package for networking utilities** and the comment in the bazel file to **This file contains the dependecies needed by the files under the networking package to run**. Can you please tell me what to change?

I was refering to the comment in the BUILD.bazel file.

**Comment by fsharpasharp** [comment]
> This file contains the dependecies needed by the files under the networking package to run.

This phrase describes the configuration itself. We want the comment in the BUILD.bazel file to describe the package (the directory). I was refering to this comment when I was talking about the description of the package.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp I made the change is it good now?

**Comment by fsharpasharp** [comment]
> @fsharpasharp I made the change is it good now?

Yes, that looks good. I'll wait for the checks, and then I'll approve the PR.

**Comment by jonathanalvares9009** [comment]
@fsharpasharp PR has passed all checks

**Review by fsharpasharp — APPROVED** [review]
Nice, thanks!

**Comment by jonathanalvares9009** [comment]
@BenHenning please review my PR

**Review by BenHenning — APPROVED** [review]
Thanks @jonathanalvares9009! This LGTM.

**Pull request merged**